### PR TITLE
create a new object version before running assemblyWF

### LINKED ISSUE #40: Pass a version to create_workflow_by_name
https://github.com/sul-dlss/lyberservices-scripts/blob/7464f87696391bd664d4acc45eef189abc28c85e/lib/pre_assembly/digital_object.rb#L472

It's deprecated to not pass a version.

### LINKED ISSUE #44: pre_assemble script needs to interact cleanly with versioningWF
**Describe the bug**

see sul-dlss/argo#1781

To run an object through the lyberservices-scripts, the user needs to manually open a version in Argo (?).   However, when lyberservices-scripts run `bundle.pre_assemble`, they kick off `assemblyWF` and `accessioningWF`, which can complete, but then the `versioningWF` needs to be manually set to completed in Argo.

**User Impact**

PITA for Andrew or anyone else who uses this code for object assembly and accessioning.

### PR DESCRIPTION
**Pull request by ndushay**
~~[HOLD for Andrew to test in stage and sign off]~~ (Andrew approved)

## Why was this change made?

When these scripts are used for the assemblyWF, if an object is being re-run (not the first version) then the user has needed to manually open a version in argo for each object, and then when the script finishes running, the user then needs to manually close version for each of the objects.

@andrewjbtw with this change, the version opening is also included, so these will work like pre-assembly scripts.

Fixes #44 ; This is the lyberservices-scripts part of sul-dlss/argo/issues/1781.
This PR also fixes #40 - pass the object version to workflows client `create_workflow_by_name`

## Was the usage documentation (e.g. README, consul, DevOpsDocs, wiki) updated?

n/a

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 dist: bionic
 language: ruby
 cache: bundler
-env:
-  global:
-    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up bundle install
```
Comment (jcoyne) [inline .travis.yml]: why is this removed?

**Comment by andrewjbtw** [comment]
@jcoyne the primary use for this script at this point is being able to control publish/shelve/preserve settings. Regular preassembly sets defaults that do not always match actual needs and this script is less painful than all other ways we currently have of setting publish/shelve/preserve permissions.

**Comment by jcoyne** [comment]
@andrewjbtw I’d love to see if this is a case that could be handled by the sdr client gem.

**Comment by ndushay** [comment]
Had to fix stage with PR #48 first.  @andrewjbtw will test this.

@jcoyne apparently there are a couple of corner cases that preassembly can't cope with at this time.   This code base is old and yucky, agree.

File: .travis.yml (hunk 1 above)
Comment (ndushay) [inline .travis.yml]: this is a duplicate entry - it occurs lower in this file
